projects/validation/src/public-api.ts

```diff
@@ -1,4 +1,6 @@
 export * from './modules/email-validation/email-validation.module';
+export {SkyEmailValidationDirective as λ1} from './modules/email-validation/email-validation.directive';
 export * from './modules/url-validation/url-validation.module';
+export {SkyUrlValidationDirective as λ2} from './modules/url-validation/url-validation.directive';
```

Comment:
I know you probably didn't do this, but could we take this opportunity to put the comment above the component exports for consistency's sake?

Reply:
See: https://github.com/blackbaud/skyux-core/blob/5.0.0-next/projects/core/src/public-api.ts#L70-L74

Reply:
Done.